Tribler/Main/vwxGUI/list.py

```diff
@@ -2338,6 +2741,8 @@ def selectTab(self, tab):
         elif tab == 'my_files':
             itemKey = 4
         elif tab == 'videoplayer':
+            itemKey = 6
+        elif tab == 'creditmining':
```

Comment:
Minor detail: it's a little nicer to swap the `elif` statements (so the `itemKey`s are ascending).